packages/cf.js/src/provider.ts

```diff
@@ -87,6 +91,26 @@ export class Provider {
     });
   }
 
+  async getOrCreateAppInstance(
+    id: AppInstanceID,
+    info?: AppInstanceInfo
+  ): Promise<AppInstance> {
+    if (!this.appInstances[id]) {
```

Comment:
@mvanderh How do you feel about flipping the condition so the behavior is better reflected in the function's name?

```ts
if (id in this.appInstances) {
  return this.appInstances[id];
}

// Rest of the code is the "createAppInstance" part...
```

Reply:
I don't think that works in Javascript? Or is it new?

Reply:
@mvanderh It's part of [ES5](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in), it should work both in Node and the browser:

![image](https://user-images.githubusercontent.com/118913/49757186-8e751100-fc9a-11e8-943f-c7fd6f26f510.png)

![image](https://user-images.githubusercontent.com/118913/49757230-b1072a00-fc9a-11e8-9113-b96c72da8029.png)

Reply:
Oooh cool, TIL. I like your suggestion then

Reply:
I actually meant to reverse the code blocks, not only to apply the `in` operator :P But don't worry, let's merge this PR and deal with that later :)